### DISCUSSION TITLE
Regarding analysis options available in Instant Clue

### DISCUSSION THREAD

**Discussion by rajat893** [hnolCol/instantclue#78] [Q&A] [answered]
Hello,
Nice tool and seems very useful.
I am new to this and have been watching the YouTube tutorials for PCA and cluster analysis (https://www.youtube.com/watch?v=bsSAqHOzq
![Screenshot 2023-09-26 at 1 54 42 PM](https://github.com/hnolCol/instantclue/assets/146059027/a2b48c86-6d6b-43ed-abbf-51d6937d32ed)

However, the analysis options shown in the video for selection (Dimension reduction and cluster analysis) are not available in my Instantclue analysis panel on Mac.
<img width="235" alt="Screenshot 2023-09-26 at 1 55 25 PM" src="https://github.com/hnolCol/instantclue/assets/146059027/e11bead5-8ebb-4cc8-b7dc-b3d5aaa2bbc2">

Would really appreciate suggestions on fixing this problem.
Thank you.

**Comment by hnolCol** [comment, marked as answer]
Hi, 

please excuse the late reply! 

The Principal Component Analysis is now located in the context menu (only available for Numeric Floats). 

1. Select the columns that you want to use. 
2. Right click -> Value Transformation -> Dimensional Reduction 

You can calculate the projection and/or the drivers (evs). 
Please check the screenshot below 

![image](https://github.com/hnolCol/instantclue/assets/24191910/eec6b0e4-2bb9-45b2-bdfb-6504027e6dc1)

I really have to update the videos! If you want to change the number of components, please go to the bottom-right -> Settings -> Dimensional reduction.

Hope this helps, and please excuse again the poor response time.

**Comment by rajat893** [comment]
Thanks a lot for the reply.
This is indeed very helpful..!!
